tests/unit/ai-requirement-generator.test.tsx

```
@@ -18,12 +18,16 @@ const translate = Object.assign(
       imageErrorCount: 'You can attach up to {max} images.',
```

Biome (2.5.1)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)